flaskphiid/annotation.py

```diff
@@ -195,10 +197,26 @@ def split_annotations_by_subtypes(self):
             running_annos = []
             for anno in self.source_annotations:
                 if not running_annos or anno.type == running_annos[-1].type:
+                    '''
+                    if we have a new annotation run, 
+                    OR 
+                    the current annotation type is the same as the previous
+                    add the current annotation to the run
+                    '''
                     running_annos.append(anno)
                     continue
+
+                if anno.origin.lower() != 'compmed' and running_annos[-1].origin.lower() == 'compmed':
+                    '''
+                    if the previous run was CompMed, and the current annotation is not
+                    Then throw out the previous run
+                    '''
+                    running_annos = [anno]
```

Comment:
I'm wondering what cases this would cover well...
would this wipe out if we had for example:

SCCA Sarcoma Clinic 123 Fairview Ave, room 604
SCCA - hutchner says hosp_name
Sarcoma Clinic - hutchner says specialty
123 Fairview Ave - compmed says address
room 604 - hutchner says ward

Reply:
at the 123 Fairview Ave wouldn't it take the running_anno back to just the "hosp_name" label?

Reply:
Emily
hutchner says "PATIENT_OR_FAMILY_NAME"
compmed says "COMPMED"